Formula/drogon.rb

```diff
@@ -1,8 +1,10 @@
 class Drogon < Formula
   desc "HTTP(S) Web Application Framework (C++14/17 based)"
   homepage "https://drogon.docsforge.com"
-  url "https://github.com/drogonframework/drogon/archive/refs/tags/v1.7.2.tar.gz"
-  sha256 "a682fafc3619ba69d3d9278a80f6a386d1effcc3692b47e976de7cf9ff89d6b8"
+  # pull from git tag to get submodules
+  url "https://github.com/drogonframework/drogon.git",
```

Comment:
Shouldn’t this point to `trantor`?

Reply:
No, because homebrew will automatically do `git submodule update --init` as long as we configure `.gitmodules`, meaning that `trantor` will also be downloaded when `drogon` is git cloned.